core/mixins/_media-queries.scss

```diff
@@ -97,3 +105,16 @@
     @content;
   }
 }
+
+
+/**
+ * Retina.
+ */
+
+@mixin respond-retina($resolution: $retina-resolution) {
```

Comment:
>  I'm just not a huge fan of using terms invented by companies, in this case Apple.

The name is based off the existing `retina-bg-image()`, so I don't think it would be a good idea to have 2 different naming conventions for this. Ideally, I'd use `hidpi` like [Bourbon](http://bourbon.io/docs/#hidpi-media-query) but the sad fact is that most people think of `retina`; it's ubiqitous.